python/valis/routes/target.py

```diff
@@ -182,6 +183,7 @@ async def get_target_altid(self,
     @router.get('/spectra/{sdss_id}', summary='Retrieve a spectrum for a target sdss_id',
                 dependencies=[Depends(get_pw_db)],
                 response_model=List[SpectrumModel])
+    @valis_cache()
```

Comment:
similarly can we call the cache namespace here `target`?

Reply:
Done